### PR TITLE
adding further reading page

### PR DESCRIPTION
**Pull request by rubyimrie**
nuxt 3 version of nuxt 3 page, have used tailwind rather than raw css as suggested

### CODE REVIEW

**Comment by rubyimrie** [comment]
have removed mb-3 from headings as once merged these will be inlcuded globally, but may not look good until merged

File: pages/further-reading.vue [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center">Further Reading</h1>
+    <p class="text-justify">
+      The following list of works, curated by Professor Julian Goodare, a
```
Comment (Sephster) [inline pages/further-reading.vue]: It might be good to link to Julian's ed.ac.uk profile here: https://www.ed.ac.uk/profile/julian-goodare

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/further-reading.vue (hunk 1 above)
Comment (rubyimrie) [reply pages/further-reading.vue]: good idea, done thank you

Comment (rubyimrie) [reply pages/further-reading.vue]: do you think all the links should open in a new tab or fine as is?

Comment (Sephster) [reply pages/further-reading.vue]: Ideally they'd open in a new tab as they are external but we'd need to add some kind of visual indicator to show this is an external link then go through the whole site to add to others. I don't think we have time for this so best leave it for now

Comment (Sephster) [reply pages/further-reading.vue]: oh I see you've made it a new tab/browser link. That's fine as well. Happy to merge.

**Review by Sephster — APPROVED** [review]

**Pull request merged**